### ISSUE TITLE
YouTube link rendering as normal inline code

### ISSUE THREAD

**Issue by FilledStacks** [markhoney/gridsome-plugin-remark-youtube#1]
I'm trying to use the format as described in the docs but my video still renders as normal markdown code.

My Markdown

    `youtube:https://youtu.be/fZuLh-oc5Ao`

gridsome.config.js

```javascript
// This is where project configuration and plugin options are located. 
// Learn more: https://gridsome.org/docs/config

// Changes here requires a server restart.
// To restart press CTRL + C in terminal and run `gridsome develop`

module.exports = {
  siteName: 'FilledStacks',
  siteDescription: 'Flutter and Mobile development tutorials and guides',

  plugins: [
    {
      // Create posts from markdown files
      use: '@gridsome/source-filesystem',
      options: {
        typeName: 'Post',
        path: 'content/posts/*.md',
        route: '/:slug',
        refs: {
          // Creates a GraphQL collection from 'tags' in front-matter and adds a reference.
          tags: {
            typeName: 'Tag',
            route: '/tag/:id',
            create: true
          }
        },
        remark: {
          plugins: [
            ['gridsome-plugin-remark-youtube']
          ]
        }
      }
    }
  ],

  transformers: {
    //Add markdown support to all file-system sources
    remark: {
      externalLinksTarget: '_blank',
      externalLinksRel: ['nofollow', 'noopener', 'noreferrer'],
      anchorClassName: 'icon icon-link',
      plugins: [
        '@gridsome/remark-prismjs',
      ]
    }
  },
}
```

It might be because of the other remark config but I have tried to put the plugin inside the transformers section and also outside the options. Neither of those two things works.

**Comment by FilledStacks** [comment]
I removed the plugin and used an embed url instead. I add an embedded youtube url into my post frontmatter section and then use that to render in my post view. Couldn't get the plugin to work. Gonna leave this open because it does seem like a glaring bug.

**Comment by narration-sd** [comment]
Having the same result this afternoon -- no variations appear to function...?

For the current case, better anyway to use a <video ...></video> tag in the markdown, with a live mp4. This works well.

A point is that you need to use this expanded form, <video ...></video>, rather than <video ... />, or remark will eliminate anything following in the .md file.

**Comment by FilledStacks** [comment]
@narration-sd Thanks. I didn't even use the <video > tag. For now I just added the videoLink as apart of my frontmatter and use that to show in my header. When I need to add videos in between my content I'll keep this in mind.

**Comment by narration-sd** [comment]
@FilledStacks That;s kind of how I thought you'd done, and got yourself on the air. I actually think there's a lot of opportunity for plugins of this nature. 

To have that, they'll need to keep up with Gridsome so they're reliable in working, and also allow the range of options we'd expect for the content -- preload, poster, etc. for video, and ease of strapping in css tags and format systems such as Flex and Grid. 

One can see this coming, for what you'd really want to do with Markdown content - in Gridsome, a very nice way of extending  on that, without altering Markdown itself.

**Comment by FilledStacks** [comment]
@narration-sd yes. Lot's of opportunities here for plugins, it's also pretty simple to implement. I might tackle this exact plugin and make it more reliable. Or just fix and submit a pull request to fix the issue of it not working.
